kstreams-live-update/example-db/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM debezium/example-mysql:0.10
```

Comment:
Can we keep both? I.e. rename _example-db_ to _example-db-mysql_, and leave otherwise as is. Then add _example-db_postgres_ with your version. Then add a new service to the Compose file for PG, next to MySQL (alternatively, have two different Compose files). The Compose file would inject one of the DB names as env var into the event-source service.

Reply:
Done.